pkg/virt-handler/vm.go

```diff
@@ -878,6 +878,10 @@ func (c *VirtualMachineController) updateHotplugVolumeStatus(vmi *v1.VirtualMach
 				volumeStatus.Reason = VolumeUnMountedFromPodReason
 			}
 		}
+	} else if len(vmi.Status.MigratedVolumes) > 0 && vmi.Status.MigrationState != nil && c.host == vmi.Status.MigrationState.SourceNode &&
+		volumeStatus.Phase == v1.VolumeReady {
+		needsRefresh = true
```

Comment:
If I understand this correctly, by saying `needsRefresh = true` we will now be reconciling a VMI with hotplug volumes every second until the migration completes. Is that correct? Can you help me understand why?